apps/epic-react/src/templates/lesson-template.tsx

```diff
@@ -333,6 +393,7 @@ const ExerciseTemplate: React.FC<{
                     >
                       <LessonList
                         module={module}
```

Comment:
```suggestion
                        module={epicReactModule}
```
I must have missed this with the merge conflicts I was getting but we want to pass `epicReactModule` so the `moduleType` is always 'module'

Reply:
Fixed